src/icons/iconController.tsx

```diff
@@ -34,3 +34,26 @@ export const GetIcons = (knowledge: boolean, className?: string): JSX.Element[]
         <M5 key={5} className={className} />
     ];
 };
+
+export const GetIcon = (knowledge: boolean, level: number): JSX.Element => {
+    if(knowledge) {
+        switch (level) {
+            case 0: return <K0/>
+            case 1: return <K1/>
+            case 2: return <K2/>
+            case 3: return <K3/>
+            case 4: return <K4/>
+            case 5: return <K5/>
+            default: return <div></div>
```

Comment:
Return empty div can create invisible problems, better to change to Fragent
```suggestion
            default: return <Fragment />
```

Reply:
needs import of Fragment btw